kustomizations/apps/data-hub-configs/s3-csv-data-pipeline.config.yaml

```diff
@@ -316,3 +316,18 @@ s3Csv:
       objectName: "airflow-config/s3-csv/{ENV}-state/sciety_events.json"
     recordProcessingSteps:
       - parse_json_value
+    airflow:
+      taskParameters:
+        queue: 'kubernetes'
+        executor_config:
+          pod_override:
+            spec:
+              containers:
+                - name: 'base'
```

Comment:
We were not quite sure whether this structure is correct or whether the name should be something different. But it seems to match some examples.